libtrip-notification/src/test/java/com/mapbox/navigation/trip/notification/MapboxTripNotificationTest.kt

```diff
@@ -1,20 +1,43 @@
 package com.mapbox.navigation.trip.notification
 
-import org.junit.Assert
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.pm.PackageManager
+import android.content.res.Configuration
+import android.content.res.Resources
+import android.text.format.DateFormat
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import java.util.Locale
 import org.junit.Before
-import org.junit.Test
 
 class MapboxTripNotificationTest {
```

Comment:
There are no actual tests in this class. Hopefully, we can abstract android components enough to test the logic.